client/flutter/lib/home_page.dart

```diff
@@ -100,6 +105,29 @@ class HomePage extends StatelessWidget {
       ),
     );
   }
+
+  Future<void> _tryShareOrShowSnackBar(BuildContext context) async {
+    const url = 'https://www.who.int/covid-19-app';
+    const shareText = 'Check out the official COVID-19 Guide App $url';
+
+    try {
+      await Share.share(shareText, subject: 'Official COVID-19 Guide App');
```

Comment:
Can we remove the subject for now:

```suggestion
      await Share.share(shareText);
```